src/index.html

```diff
@@ -13,10 +17,15 @@
     <title>Stop watch</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.css"
     />
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <div class="stopwatch stopwatch--speed-up">
+      <div class="stopwatch__center"></div>
+      <div class="stopwatch__hand stopwatch__hand--seconds"></div>
+      <div class="stopwatch__hand stopwatch__hand--minutes"></div>
```

Comment:
It's generally a good practice to add some content or at least aria-labels to the empty tags for better accessibility. Screen readers will then be able to correctly announce the content.